content/blog/2022-07-25_Galaxy_Climate_Project_Introduction_Blog_2/index.md

```diff
@@ -0,0 +1,143 @@
+The very first diagnostic step began in the early 19th century when ice ages and other natural changes in paleoclimate were first suspected and the natural greenhouse effect was first identified. Since then, there have been researches. There are some very important satellites constantly orbiting the earth and collecting data with respect to climate variables such as aerosols, ocean temperature, ocean color, albedo, ocean chlorophyll content, and thousands of others. The category of satellites used to monitor short-term weather patterns, climate change, and natural disasters are called [Earth Observation Satellites](https://en.wikipedia.org/wiki/List_of_Earth_observation_satellites).
+
+
+Some of the important satellites that are revolutionary in the field of climate change are [MERRA](https://gmao.gsfc.nasa.gov/reanalysis/MERRA-2/), [MODIS AQUA](https://oceancolor.gsfc.nasa.gov/data/aqua/), [MODIS TERRA](https://terra.nasa.gov/about/terra-instruments/modis), [LANDSAT 8](https://landsat.gsfc.nasa.gov/satellites/landsat-8/) , [GLDAS](https://ldas.gsfc.nasa.gov/gldas), [SENTINEL-1B](https://ldas.gsfc.nasa.gov/gldas) and [GOSAT](https://www.gosat.nies.go.jp/en/)
+		
+There are many new initiatives with a special focus on the coastal cities that are at a risk of submergence. The [falcon -9](https://www.spacex.com/vehicles/falcon-9/) carried the [Sentinel -6](https://www.eumetsat.int/sentinel-6#:~:text=Once%20the%20state%2Dof%2Dthe,the%20oceans%2C%20weather%20and%20coastlines.) to the Earth’s orbit . What does the Sentinel-6 satellite do?The Sentinel-6 mission is part of the Copernicus programme initiative, the main objective of the Sentinel-6 mission is to measure sea surface topography with high accuracy and reliability to support ocean forecasting systems, environmental monitoring and climate monitoring.
+
+The data these satellites record are available to the common public in the following   formats: [ASCII](https://www.xmswiki.com/wiki/SMS:ASCII_Dataset_Files_*.dat), [netCDF4](https://unidata.github.io/netcdf4-python/), [HDF5](https://docs.h5py.org/en/stable/high/dataset.html), [TIFF](https://ieee-dataport.org/data-formats/tif-1), [SHP](https://en.wikipedia.org/wiki/Shapefile), etc. There are plenty of python ad R libraries to study these datasets. One of the best and popular code-free option is  the Galaxy climate, with easy to understand UI. Thus as a part of my internship, I will be utlising this server to analyse the commonly used datasets in my region. This can ease the process to a great extent as the new user has to just upload his data and execute my workflow. I’ll be sharing the datasets I’ve used and the corresponding Jupyter notebooks of my analysis in the next blog. In the meanwhile, let’s explore the climatogram of Mumbai city and the tool I prepared.

```

Comment:
```suggestion
The data these satellites record are available to the common public in the following formats: [ASCII](https://www.xmswiki.com/wiki/SMS:ASCII_Dataset_Files_*.dat), [netCDF4](https://unidata.github.io/netcdf4-python/), [HDF5](https://docs.h5py.org/en/stable/high/dataset.html), [TIFF](https://ieee-dataport.org/data-formats/tif-1), [SHP](https://en.wikipedia.org/wiki/Shapefile), etc. There are plenty of Python ad R libraries to study these datasets. One of the best and popular code-free option is the Galaxy climate platform, with easy to understand UI. Thus as a part of my internship, I will be utlising this server to analyse the commonly used datasets in my region. This can ease the process to a great extent as the new user has to just upload his data and execute my workflow. I’ll be sharing the datasets I’ve used and the corresponding Jupyter notebooks of my analysis in the next blog. In the meanwhile, let’s explore the climatogram of Mumbai city and the tool I prepared.
```